Model/app.js

```diff
@@ -46,45 +47,59 @@ app.controller('myController', ['$scope', ($scope) => {
       };
       reader.readAsText(file[i]);
     }
+    console.log("++++++++upload", $scope.fileContent)
```

Comment:
Thanks I have made changes to the file by removing the console.log() in code base